tests/unit/test_sampler.py

```
@@ -0,0 +1,209 @@
+    
+    @pytest.mark.asyncio
+    async def test_generate_person(self, data_paths):
+        """Test person generation."""
+        # Generate a person
+        person = await generate_person(
+            gender="M",
+            min_age=25,
+            max_age=35,
+            names_path=data_paths["names_path"],
+            surnames_path=data_paths["surnames_path"],
+            middle_names_path=data_paths["middle_names_path"]
```

Ruff (0.12.2)

Use of `assert` detected

(S101)